### ISSUE TITLE
[Validation] Validate project name and video url.

### ISSUE THREAD

**Issue by theprogrammerinyou** [aadeshkulkarni/sanchay-ai-frontend#4]
Validation must be done for video URL and project name and the error must be shown to user to generate thumbnail, transcription or subscripts.

**Comment by aadeshkulkarni** [comment]
Yes, I've added it as TODO in the code. But before adding validation, need to pick a toast/notification library that's lightweight and has good support. 
Alternatively, can be implemented within the project as well.

**Comment by theprogrammerinyou** [comment]
There a couple, cogo-toast, react-toastify.React toasty is good,  its well updated. You can check that.

**Comment by aadeshkulkarni** [comment]
Alright. Let's go ahead with React-Toastify https://fkhadra.github.io/react-toastify/introduction

Whoever picks this up,
Validations needed:
1. Project name
2. Video URL (Regex check to see if it's a valid url)
3. Atleast 1 of the 3 checkboxes should be true

If all of the above valid, allow API call. Else, show toast notification.

**Comment by theprogrammerinyou** [comment]
Can i pick this up?

**Comment by aadeshkulkarni** [comment]
Yes
